docs/intro/install.rst

```diff
@@ -1,245 +1,56 @@
 .. _intro-install:
 
-==================
-Installation guide
-==================
+============
+Installation
+============
 
-Installing Scrapy
-=================
+Scrapy supports Python 3.5 or above under CPython (the default Python
+implementation) and PyPy (starting with PyPy 5.9).
 
-Scrapy runs on Python 3.5 or above
-under CPython (default Python implementation) and PyPy (starting with PyPy 5.9).
-
-If you're using `Anaconda`_ or `Miniconda`_, you can install the package from
-the `conda-forge`_ channel, which has up-to-date packages for Linux, Windows
-and OS X.
-
-To install Scrapy using ``conda``, run::
+.. _install-pypi:
+.. _intro-install-platform-notes:
+.. _intro-install-ubuntu:
+.. _intro-install-macos:
 
-  conda install -c conda-forge scrapy
+Installing from the Python Package Index
+========================================
 
-Alternatively, if you’re already familiar with installation of Python packages,
-you can install Scrapy and its dependencies from PyPI with::
+You can install Scrapy from the `Python Package Index`_::
 
     pip install Scrapy
 
-Note that sometimes this may require solving compilation issues for some Scrapy
-dependencies depending on your operating system, so be sure to check the
-:ref:`intro-install-platform-notes`.
-
-We strongly recommend that you install Scrapy in :ref:`a dedicated virtualenv <intro-using-virtualenv>`,
-to avoid conflicting with your system packages.
-
-For more detailed and platform specifics instructions, as well as
-troubleshooting information, read on.
-
-
-Things that are good to know
-----------------------------
+For more information, see :ref:`packaging:installing-packages`.
```

Comment:
I wonder if there is a more beginner-friendly document available. This one is good because it is official, but it can be scary. Consider how it starts:

> This section covers the basics of how to install Python packages.

And then, the next paragraph:

> It’s important to note that the term “package” in this context is being used as a synonym for a distribution (i.e. a bundle of software to be installed), not to refer to the kind of package that you import in your Python source code (i.e. a container of modules). It is common in the Python community to refer to a distribution using the term “package”. Using the term “distribution” is often not preferred, because it can easily be confused with a Linux distribution, or another larger software distribution like Python itself.

Not exactly the information beginners are looking for.